.github/workflows/test.yml

```diff
@@ -0,0 +1,18 @@
+name: Validate JSON Schema files
+on: [push]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: Install jsonschema
+        run: pip install jsonschema==3.2.0
```

Comment:
Didn't manage to run this using the latest jsonschema version on my machine or on GitHub Actions

Looking [at releases](https://pypi.org/project/jsonschema/#history) and issues it seems like the v4 is not mature enough.

Reply:
I used a straightforward `pip install` here instead of adding a `requirements.txt` etc. Let me know if you're okay with that.